components/board/pi/common/common.go

```diff
@@ -17,12 +17,13 @@ var ModelName = resource.NewDefaultModel("pi")
 
 // ServoConfig is the config for a pi servo.
 type ServoConfig struct {
-	Pin       string   `json:"pin"`
-	Min       int      `json:"min,omitempty"`
-	Max       int      `json:"max,omitempty"`
-	StartPos  *float64 `json:"starting_position_degs,omitempty"`
-	HoldPos   *bool    `json:"hold_position,omitempty"` // defaults True. False holds for 500 ms then disables servo
-	BoardName string   `json:"board"`
+	Pin         string   `json:"pin"`
+	Min         int      `json:"min,omitempty"`
+	Max         int      `json:"max,omitempty"`
+	StartPos    *float64 `json:"starting_position_degs,omitempty"`
+	HoldPos     *bool    `json:"hold_position,omitempty"` // defaults True. False holds for 500 ms then disables servo
+	BoardName   string   `json:"board"`
+	MaxRotation int      `json:"max_rotation_deg,omitempty"`
```

Comment:
Can you add a comment here to describe what this is. Particularly to help differentiate it from `Max`. 
As far as I can tell, this must be set based on the data sheet for all servos that aren't 180deg. @randhid is that right?

Reply:
Yes. Servos that don't have a 0-180 degree range need this parameter to be set. Note: continuous servos are not supported by the `pi` model.